chall01/hhuhtane.py

```diff
@@ -0,0 +1,37 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    hhuhtane.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: hhuhtane <hhuhtane@student.hive.fi>        +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/03/26 15:47:35 by hhuhtane          #+#    #+#              #
+#    Updated: 2020/03/26 16:51:09 by hhuhtane         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+import sys
+
+alphabet = "abcdefghijklmnopqrstuvwxyz"
+capital = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+arg_i = 0
+i = 0
+morse = ""
+codes = ['.-', '-...', '-.-.', '-..', '.', '..-.', '--.', '....', '..', '.---',
+         '-.-', '.-..', '--', '-.', '---', '.--.', '--.-', '.-.', '...', '-',
+         '..-', '...-', '.--', '-..-', '-.--', '--..']
+
+if len(sys.argv) == 2:
+    while arg_i < len(sys.argv[1]):
+        i = 0
+        while i < 26:
+            if alphabet[i] == sys.argv[1][arg_i]:
+                morse += codes[i]
+            if capital[i] == sys.argv[1][arg_i]:
+                morse += codes[i]
```

Comment:
I need to check your code but I don't really see where you check there isn't another non alpha character.

You could have create a Python Dictionary : https://www.tutorialspoint.com/python3/python_dictionary.htm

```python
{'a': '.-', 'b': '-...', 'c': '-.-.', 'd': '-..', 'e': '.', 'f': '..-.', 'g': '--.', 'h': '....', 'i': '..', 'j': '.---', 'k': '-.-', 'l': '.-..', 'm': '--', 'n': '-.', 'o': '---', 'p': '.--.', 'q': '--.-', 'r': '.-.', 's': '...', 't': '-', 'u': '..-', 'v': '...-', 'w': '.--', 'x': '-..-', 'y': '-.--', 'z': '--..'}
```

and then do something like 
```python
morse += mydict[arg_i]
```